test/integration/propControls/index.spec.ts

```diff
@@ -1,18 +1,23 @@
 import { test, expect } from '@playwright/test';
-import { ToolpadHome } from '../../models/ToolpadHome';
+import invariant from 'invariant';
+import * as path from 'path';
 import { ToolpadEditor } from '../../models/ToolpadEditor';
 import clickCenter from '../../utils/clickCenter';
-import domInput from './domInput.json';
+import { createApplication } from '../../utils/toolpadApi';
+import { readJsonFile } from '../../utils/fs';
 
 test('can control component prop values in properties control panel', async ({
   page,
   browserName,
+  baseURL,
 }) => {
-  const homeModel = new ToolpadHome(page);
+  invariant(baseURL, 'playwright must be run with a a baseURL');
+
+  const dom = await readJsonFile(path.resolve(__dirname, './domInput.json'));
```

Comment:
I think we're using `readJsonFile` in some tests but just importing the JSON file in others?
Should we do it in a single consistent way across all tests?

Reply:
We should use `readJsonFile` for this. There is not much benefit in having it statically typed, and it will overwhelm the type checker

Reply:
Ok, we can change everything to using `readJsonFile` then, no need for it to be in this PR though.